xbmc/platform/android/GPUInfoAndroid.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "GPUInfoAndroid.h"
+
+std::shared_ptr<CGPUInfo> CGPUInfo::GetGPUInfo()
+{
+  return std::make_shared<CGPUInfoAndroid>();
+}
+
+bool CGPUInfoAndroid::SupportsPlatformTemperature() const
```

Comment:
a paticular reason why this function (and the others) is/are not implemented in the header?

Reply:
Just leaves the file in place for later implementation if that's the case. Any reason why it shouldn't?

Reply:
> Any reason why it shouldn't?

no, fine